lib/Mojolicious/Plugin/WebSocketProxy.pm

```diff
@@ -0,0 +1,278 @@
+package Mojolicious::Plugin::WebSocketProxy;
+
+use Mojo::Base 'Mojolicious::Plugin';
+use Mojo::WebSocketProxy::Dispatcher::Config;
+use Mojo::WebSocketProxy::Dispatcher;
+our $VERSION = "0.06";
```

Comment:
In the readme it is stated that vesion is 0.06 :) use pod2mkdown to keep them in synch